frontend/src/components/custom-tools/prompt-card/OutputForIndex.jsx

```diff
@@ -38,8 +149,52 @@ function OutputForIndex({ chunkData, setIsIndexOpen, isIndexOpen }) {
       footer={null}
       width={1000}
     >
+      <div className="chunk-search-container">
+        <Input
+          placeholder="Search..."
+          value={searchTerm}
+          onChange={handleSearchChange}
+          className="chunk-search-input"
+        />
+        <div className="search-control-container">
+          <Button
+            size="small"
+            onClick={handlePrev}
+            disabled={highlightedChunks.length === 0}
```

Comment:
@jagadeeswaran-zipstack Please make sure that we use optional chaining to avoid potential `TypeError` issues.

Reply:
highlightedChunks is a array so we won't face TypeError. Anyways I'll add that.

Reply:
Thanks for clarifying, @jagadeeswaran-zipstack. Please make this change in other places as well, wherever applicable.